test/passes/translate-to-fuzz_all-features.txt

```diff
@@ -1,24 +1,23 @@
 (module
  (type $none_=>_none (func))
+ (type $funcref_f64_=>_i32 (func (param funcref f64) (result i32)))
```

Comment:
We should really replace this test with something that's more precise about what it is we're actually testing.

Reply:
This test could possibly just print out the metrics, and not the actual body. Seems like that's good enough I can do that separately.